typescript-selenium/driver/Driver.ts

```diff
@@ -0,0 +1,14 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import { ThenableWebDriver } from "selenium-webdriver";
+
+export interface Driver {
+    get(): ThenableWebDriver
```

Comment:
What is the reason to return `Thenable` here?
What about returning the driver object itself and await for it in appropriate driver implementation?

Reply:
@Ohrimenko1988 Is there a reason for this?